src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -232,6 +238,12 @@ export class OperationDetails {
         if (operation) {
             await this.loadDefinitions(operation);
             if (this.showExamples) this.parseExamples(operation);
+
+
+            if(!operation) return null;
+
+            this.loadExamples(operation);
```

Comment:
Why load examples is after parseExamples?

Reply:
parse examples actually also does operations over the examples from the response. Let me rename it thanks

Reply:
updated thank you